src/algorithms.jl

```diff
@@ -5225,20 +5230,39 @@ struct SofSpa10 <: OrdinaryDiffEqPartitionedAlgorithm end
 struct IRKN3 <: OrdinaryDiffEqPartitionedAlgorithm end
 
 """
+A 4th order explicit Runge-Kutta-Nyström method which can be applied directly on second order ODEs. Can only be used with fixed time steps.
+
+In case the ODE Problem is not dependent on the first derivative consider using Nystrom4VelocityIndependent to increase performance.
```

Comment:
```suggestion
In case the ODE Problem is not dependent on the first derivative consider using 
[`Nystrom4VelocityIndependent`](@ref) to increase performance.
```
This let's Documenter add a reference/link.